cypress/utils/commands.js

```diff
@@ -3,6 +3,126 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { BASE_PATH } from './constants';
+
+export const ADMIN_AUTH = {
+  username: Cypress.env('username'),
+  password: Cypress.env('password'),
+  set newUser(changedUsername) {
+    this.username = changedUsername;
+  },
+  set newPassword(changedPassword) {
+    this.password = changedPassword;
+  },
+};
+
+export const CURRENT_TENANT = {
+  defaultTenant: 'private',
+  set newTenant(changedTenant) {
+    this.defaultTenant = changedTenant;
+  },
+};
+
+export const supressNoRequestOccurred = () => {
+  cy.on('fail', (err) => {
+    if (err.message.includes('No request ever occurred.')) return false;
+  });
+};
+
+// TODO: Add commands to ./index.d.ts for IDE discoverability
+
+/**
+ * This overwrites the default visit command to authenticate before visiting
+ * webpages if SECURITY_ENABLED cypress env var is true
+ */
+Cypress.Commands.overwrite('visit', (orig, url, options) => {
```

Comment:
Have we validated these overrides?

Reply:
I'm still validating/fixing issues for running security enabled tests and its overrides. But as you said should not be a blocker for this PR or to run local CI.